### DISCUSSION TITLE
Fails to recognize my quotation marks

### DISCUSSION THREAD

**Discussion by crashingbooth** [aradzie/keybr.com#338] [Q&A] [answered]
Is there anything I need to do to get it to recognize quotation marks? All other characters/punctuation seem to work fine but my quotes are ignored as input. All I can do is reload the page whenever I see them in the text.

Using a Sonoma 14.3.1 with a Canadian keyboard
• I've tried switching the keyboard to British and Japanese/Romaji, no effect
• have enabled and disabled smart quotes in the keyboard settings, again no effect

**Comment by aradzie** [comment]
What are the exact quotation marks? 
Can you please open any text editor, type those quotation marks, then paste them here?

Also, if you type those marks on this page, https://domeventviewer.com/key-event-viewer.html what events do you see?

Here's for example how it works for me:

![image](https://github.com/user-attachments/assets/20cd7405-1c7f-4f9a-b999-3073a5babc22)

**Comment by aradzie** [comment]
Also, please try disabling the "Stop cursor on error" option:

![image](https://github.com/user-attachments/assets/c7003a64-bd7a-4ea7-97eb-926d38f119e5)

Then you'll be able to see what you are actually typing.

**Comment by crashingbooth** [comment]
Thanks for the reply.
Here is my quote C&Ped from vs code: "

here is what is displayed in the keyboard event viewer: 
<img width="1181" alt="Screenshot 2024-11-29 at 1 06 38 PM" src="https://github.com/user-attachments/assets/56a93d48-432e-4698-aa56-56539c81a7da">

and when I disable stop cursor on error, it seems that it isn't misidentifying the keypress (ie it doesn't treat it as an error), it simply doesn't acknowledge it as correct or incorrect. I can see the shift and quote keys turn black in the keyboard graphic beneath the text while the key presses are happening

**Comment by aradzie** [comment, marked as answer]
Hmm, your browser reports invalid input events:

![annotely_image](https://github.com/user-attachments/assets/238b6e90-6ce4-4673-aec6-67583fe45317)

See, that row highlighted in red, number three. There is no data in the input event, the `data` column is empty.

Compare with [my picture](https://github.com/aradzie/keybr.com/discussions/338#discussioncomment-11417437), rows number 2 and 6. The `inputType` column is correctly set to `inputText` and the `data` column contains some characters.

Something is not right with your browser. And I'm afraid I cannot do anything about it.

Maybe you have a browser extension which interferes with keybr?

**Comment by crashingbooth** [comment]
actually good call about the browser extension. I have 'ProKeys' which I installed years ago and basically forgot about. Disabling it resolves the issue. Thanks for your time!
